packages/client/tests/functional/0-legacy-ports/batch-find-unique/tests.ts

```diff
@@ -76,7 +78,47 @@ testMatrix.setupTestSuite(() => {
       }
     })
 
-    expect(executedBatchQuery).toMatchSnapshot()
```

Comment:
Alternatively, I think snapshots can be named via the first optional argument. That said, I kinda prefer the inline version.